includes/compat/class-wc-stripe-subs-compat.php

```diff
@@ -220,6 +222,37 @@ public function process_subscription_payment( $amount = 0.0, $renewal_order, $re
 
 			$order_id = WC_Stripe_Helper::is_wc_lt( '3.0' ) ? $renewal_order->id : $renewal_order->get_id();
 
+
+			// Early renewals should be treated like normal payments, but with a different output.
```

Comment:
Please clarify what do you mean by "normal payments" here (a few words about off-session vs on-session would help)

Reply:
Done in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1084/commits/8b6e943f5ffd40cdbd5cf0a7e37abbbbcfc81dde